python/res/fm/rms/rms_run.py

```diff
@@ -25,6 +25,7 @@ class RMSRun(object):
     _multi_seed_file = "random.seeds"
     _max_seed = 2146483648
     _seed_factor = 7907
+    BACKUP_ENVIRONMENT_VARIABLE = "_PRE_RMS_PYTHONPATH"
```

Comment:
You are setting this constant that you're only using once. This can just be an inline string literal whenever you need to use it. We're won't be changing the name of the backup envvar, so why have an unnecessary level of indirection.

Reply:
I'm not very opinionated in this situation, so I will change it. But in general I think it's nice to collect "magic"/constant strings in one place!